src/CodeGen.fs

```diff
@@ -1072,40 +1076,130 @@ let sampleFSharpClientMember query queryName hasVariables = stringBuffer {
       (if hasVariables then " input" else "()")
 }
 
-let sampleGraphqlClient projectName clientName errorType members = stringBuffer {
-    sprintf """namespace %s
+let sampleFSharpSystemClientMember query queryName hasVariables = stringBuffer {
+    sprintf """    member _.%sAsync(%s) =
+        async {
+            let query = %s
+
+            let inputJson = JsonSerializer.Serialize(%s, options)
+
+            let! response =
+                httpClient.PostAsync(url, new StringContent(inputJson, Encoding.UTF8, "application/json"))
+                |> Async.AwaitTask
+
+            let! responseContent = Async.AwaitTask(response.Content.ReadAsStringAsync())
+            let responseJson = JsonSerializer.Deserialize<JsonElement>(responseContent, options)
+
+            match response.IsSuccessStatusCode with
+            | true ->
+                let errorsReturned =
+                    match responseJson.TryGetProperty ("errors") with
+                    | true, value -> value.GetArrayLength() > 0
+                    | false, _ -> false
+
+                if errorsReturned then
+                    let response = JsonSerializer.Deserialize<GraphqlErrorResponse> (responseContent, options)
+                    return Error response.errors
+                else
+                    let response = JsonSerializer.Deserialize<GraphqlSuccessResponse<%s.Query>> (responseContent, options)
+                    return Ok response.data
+
+            | errorStatus ->
+                let response = JsonSerializer.Deserialize<GraphqlErrorResponse> (responseContent, options)
+
+                return Error response.errors
+        }
+
+    member this.%s(%s) = Async.RunSynchronously(this.%sAsync%s)
+"""
+      queryName
+      (if hasVariables then "input: " + queryName + ".InputVariables" else "")
+      ("\"\"\"\n" + addLines query + "\n            \"\"\"")
+      (if hasVariables then "{ query = query; variables = Some input }" else "{ query = query; variables = None }")
+      queryName
+
+      queryName
+      (if hasVariables then "input: " + queryName + ".InputVariables" else "")
+      queryName
+      (if hasVariables then " input" else "()")
+}
```

Comment:
Try using string interpolation. I suppose we don't need `stringBuffer` here.
It is better to cache values via `let` prior to string interpolation.